src/lib/people/team/team.ts

```diff
@@ -0,0 +1,47 @@
+import * as v from 'valibot';
+
+export const committees = [
+    'Executive',
+    'Innovation',
+    'Service',
+    'External Relations',
+    'Membership & Internals',
+    'Branding & Creatives',
+    'Engineering',
+];
+
+const MemberSchema = v.object({
+    name: v.string(),
+    title: v.string(),
+    committee: v.pipe(
+        v.string(),
+        v.check(i => committees.includes(i), 'invalid committee'),
+    ),
+    socials: v.optional(v.record(v.string(), v.pipe(v.string(), v.url()))),
+});
+
+export interface Member extends v.InferOutput<typeof MemberSchema> {
+    img: string;
+}
+
+const baseRe = /(?<base>\w+).json$/;
+
+export async function getTeam(): Promise<Member[]> {
+    const imports = import.meta.glob<Member>('./members/*.json');
+
+    const promises = Object.entries(imports).map(async ([path, asset]) => {
+        // Derive the image name from the original path
+        // It goes without saying that this may not be the right approach for this
+        const base = path.match(baseRe)?.groups?.base;
+        const img: string = (await import(`$lib/people/team/images/${base}.webp`)).default;
```

Comment:
Why was the manual `string` type annotation necessary here? Shouldn't TypeScript be able to infer this anyway?

Reply:
It actually doesn't infer it, since default evaluates to `any`. This is probably because the return type of `default` (and import for that matter) depends on the file that is eventually imported (i.e. JSON -> Array/Object, image -> string).

Reply:
Oh, that's funny. I think I know why that happens. The TypeScript LSP figures out the return type for `import` based on the `declare module` declarations in the app (including `node_modules`). Vite specifically adds a `declare module` for `*?url` (i.e., all modules that end in `?url`), which allows TypeScript to infer the import as `string`. Since we only care about the URLs anyway, let's use the `?url` import instead of the usual bare path. If even that doesn't work, I have a few more tricks up my sleeve that aren't as clean. 😅

Reply:
Doing some testing, `*?url` really doesn't seem to have any effect on type inferences for vite dynamic imports, but it does work for normal static imports. Can I hear some of those other ideas?